packages/contracts-bedrock/test/dispute/DisputeGameFactory.t.sol

```diff
@@ -318,27 +318,32 @@ contract DisputeGameFactory_FindLatestGames_Test is DisputeGameFactory_Init {
     /// @dev Tests that `findLatestGames` returns the correct games, if there are less than `_n` games of the given type
     ///      available.
     function test_findLatestGames_lessThanNAvailable_succeeds() public {
+        // Need to clear out the length of the game list on forked list to avoid massive iteration.
+        if (isForkTest()) {
+            // Slot 104 is taken from the DisputeGameFactory's storage layout.
+            vm.store(address(disputeGameFactory), bytes32(uint256(104)), bytes32(0));
```

Comment:
This should work without the magic number:

```suggestion
            vm.store(address(disputeGameFactory), ForgeArtifacts.getSlot("_disputeGameList"), bytes32(0));
```

Reply:
good call :)

Reply:
Will make this change in a bit